docs/invoke.mdx

```diff
@@ -22,71 +18,701 @@ You can visualize your state machines and easily invoke actors in our drag-and-d
+
+```js
+// Parent
+actions: sendTo(
+  'childActor',
+  ({ self }) => ({
+    type: 'ping',
+    // highlight-next-line
+    sender: self
+  })
+);
+
+// Child
+actions: sendTo(
+    // highlight-next-line
+  ({ event }) => event.sender,
+  { type: 'pong' }
+)
+```
+
+In the following example, the `'client'` machine below sends the `'CODE'` event to the invoked `'auth-server'` service, which then responds with a `'TOKEN'` event after 1 second.
+
+```js
+import { createActor, createMachine, sendTo } from 'xstate';
+
+const authServerMachine = createMachine({
+  id: 'server',
+  initial: 'waitingForCode',
+  states: {
+    waitingForCode: {
+      on: {
+        CODE: {
+          // highlight-start
+          actions: sendTo(
+            ({ event }) => event.sender,
+            { type: 'TOKEN' },
+            { delay: 1000 }
+          )
+          // highlight-end
+        }
+      }
+    }
+  }
+});
+
+const authClientMachine = createMachine({
+  id: 'client',
+  initial: 'idle',
+  states: {
+    idle: {
+      on: {
+        AUTH: { target: 'authorizing' }
+      }
+    },
+    authorizing: {
+      invoke: {
+        id: 'auth-server',
+        src: authServerMachine
+      },
+      // highlight-start
+      entry: sendTo(
+        'auth-server',
+        ({ self }) => ({ type: 'CODE', sender: self })),
+      // highlight-end
+      on: {
+        TOKEN: { target: 'authorized' }
+      }
+    },
+    authorized: {
+      type: 'final'
+    }
+  }
+});
+```
+
+That specific example could be simplified by using the `sendParent(...)` action creator for the same effect. But by using `sendTo` the actors do not strictly need to be in a direct parent-child relationship to communicate.
+
+Note that by default `sendTo` will send events anonymously, in which case the reciever will not know the source of the event.
+
+:::note
+In XState v4, the `respond(...)` action creator was used for this purpose. In XState v5, use `sendTo(...)` instead.
+:::
+
+## Multiple Actors
+
+You can invoke multiple actors by specifying each in an array:
+
+```ts
+invoke: [
+  { id: 'actor1', src: 'someActor' },
+  { id: 'actor2', src: 'someActor' },
+  { id: 'logActor', src: 'logActor' }
+]
+```
+
+Each invocation will create a new instance of that actor, so even if the `src` of multiple actors are the same (e.g., `someActor` above), multiple instances of `someActor` will be invoked.
 
-- Actors can receive input. This is an event: `{ type: 'xstate.init', input: ... }`
+## Testing
 
-## Kinds of actors
+_Coming soon_
+
+## Referencing Invoked Actors
+
+Actors can be read on `snapshot.children.<actorId>[index]`. The returned value is an `ActorRef` object, with properties like:
```

Comment:
Is there a better way to describe that `snapshot.children.<actorId>` will be an array? Wasn't sure how to word this.

Reply:
Got it. Will fix.

I think I got confused by the string names from a prior XState beta version (was `checkcarvitals.DoCarVitalChecks[0]` vs `0.checkcarvitals.DoCarVitalChecks` or something)